workshops/util.py

```diff
@@ -847,9 +847,17 @@ def merge_objects(object_a, object_b, easy_fields, difficult_fields,
                 manager.set(list(related_b.all()))
 
             elif value == 'combine':
-                # remove duplicates
-                manager.add(*list(related_a.all() |
-                                  related_b.all()))
+                summed = related_a.all() | related_b.all()
+
+                # some entries may cause IntegrityError (violation of
+                # uniqueness constraint) because they are duplicates *after*
+                # being added to the database
+                for element in summed:
+                    try:
+                        with transaction.atomic():
+                            manager.add(element)
+                    except IntegrityError:
```

Comment:
Right: my question is, will catching and discarding `IntegrityError` 
mask errors that we actually want reported?  Is there a way to restrict 
this so that it only discards errors that arise when Task objects are 
duplicates?  Or am I being paranoid?